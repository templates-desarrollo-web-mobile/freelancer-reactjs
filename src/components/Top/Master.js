const Master = () => (
    <header className="masthead bg-primary text-white text-center">
            <div className="container d-flex align-items-center flex-column">
            
                <img className="masthead-avatar mb-5" src="https://startbootstrap.github.io/startbootstrap-freelancer/assets/img/avataaars.svg" alt="..." />
                
                <h1 className="masthead-heading text-uppercase mb-0">Start Bootstrap</h1>
               
                <div className="divider-custom divider-light">
                    <div className="divider-custom-line"></div>
                    <div className="divider-custom-icon"><i className="fas fa-star"></i></div>
                    <div className="divider-custom-line"></div>
                </div>
                
                <p className="masthead-subheading font-weight-light mb-0">Graphic Artist - Web Designer - Illustrator</p>
            </div>
        </header>
);

export default Master;