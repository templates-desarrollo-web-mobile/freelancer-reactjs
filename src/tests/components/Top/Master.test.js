import { shallow } from 'enzyme';
import Master from "../../../components/Top/Master";
describe('Pruebas en "Master"', () => {
    
    const wrapper = shallow(<Master/>);
    test('Debe de mostrar correctamente el componente', () => {
        
        expect( wrapper ).toMatchSnapshot();
    });

});