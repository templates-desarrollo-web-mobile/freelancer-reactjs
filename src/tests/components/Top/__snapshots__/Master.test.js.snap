// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en "Master" Debe de mostrar correctamente el componente 1`] = `
<header
  className="masthead bg-primary text-white text-center"
>
  <div
    className="container d-flex align-items-center flex-column"
  >
    <img
      alt="..."
      className="masthead-avatar mb-5"
      src="https://startbootstrap.github.io/startbootstrap-freelancer/assets/img/avataaars.svg"
    />
    <h1
      className="masthead-heading text-uppercase mb-0"
    >
      Start Bootstrap
    </h1>
    <div
      className="divider-custom divider-light"
    >
      <div
        className="divider-custom-line"
      />
      <div
        className="divider-custom-icon"
      >
        <i
          className="fas fa-star"
        />
      </div>
      <div
        className="divider-custom-line"
      />
    </div>
    <p
      className="masthead-subheading font-weight-light mb-0"
    >
      Graphic Artist - Web Designer - Illustrator
    </p>
  </div>
</header>
`;
